app/redux/ducks/classify.js

```diff
@@ -148,6 +154,14 @@ export default function reducer(state = initialState, action = {}) {
       }
       return state;
     }
+    case STORE_INTERVENTION_UUID: {
+      const { intervention } = state;
+      let lastInterventionUUID = null
+      if (intervention && intervention.hasOwnProperty('uuid')) {
+        lastInterventionUUID = intervention.uuid;
+      }
+      return Object.assign({}, state, { lastInterventionUUID });
```

Comment:
This returns a new state object even if there's no intervention. Is that intentional? I think it should return the current state in that case, but I'm not sure.